### PR TITLE
fix remove-unused-istags script

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

File: templates/remove-unused-istags.sh.j2 [hunk 1]
```diff
@@ -12,7 +12,7 @@ autorebuild_imagestream_tags=$(
 delete_threshold=$(date --date="7 days ago" --iso-8601=seconds --utc)
 
 old_imagestream_tags=$(
-    $OC get imagestreamtags -o json | \
+    $OC get imagestreamtags --chunk-size=0 -o json | \
```
Comment (lcarva) [inline templates/remove-unused-istags.sh.j2:15]: Could add a comment here to explain why this is needed?

File: templates/remove-unused-istags.sh.j2 [hunk 2]
```diff
@@ -36,4 +36,19 @@ echo
 count=$(wc -l <<<${unwanted_imagestream_tags})
 echo "SUMMARY: ${count} ImageStreamTags will be removed"
 
-$OC delete imagestreamtags ${unwanted_imagestream_tags}
+batch_limit=0
+tags_batch=''
+
+for tag in ${unwanted_imagestream_tags}; do
```
Comment (lcarva) [inline templates/remove-unused-istags.sh.j2:42]: You can simply use `xargs` to create the batches.
```bash
<<<${unwanted_imagestream_tags} xargs -n ${batch_size} $OC delete imagestreamtags
```

File: templates/remove-unused-istags.sh.j2 (hunk 1 above)
Comment (rcerven) [reply templates/remove-unused-istags.sh.j2:16]: added

File: templates/remove-unused-istags.sh.j2 (hunk 2 above)
Comment (rcerven) [reply templates/remove-unused-istags.sh.j2]: fixed

**Review by lcarva — APPROVED** [review]

**Review by MartinBasti — APPROVED** [review]

**Pull request merged**